skbio/io/format/embed.py

```diff
@@ -285,8 +288,8 @@ def _objects_to_embed(objs, fh, include_embedding_pointer=True):
                     "Embedding dimension mismatch between objects. "
                     f"({embed_fh.shape}) and ({emb.shape})"
                 )
-                if resize:
-                    embed_fh.resize(max_embsize, axis=0)
+                #if resize:
```

Comment:
Hi @paarth-b don't we want to keep this resize in place? Otherwise we are resizing the embedding objects at every iteration.